boards/arm/decawave_dwm3001c_dev/Kconfig

```diff
@@ -0,0 +1,10 @@
+# DecaWave DWM3001C DEV board configuration
+
+# Copyright (c) 2019 Stéphane D'Alu
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ENABLE_DCDC
+	bool "DCDC mode"
+	select SOC_DCDC_NRF53X_APP # todo(sahas): is this right?
```

Comment:
fixed ✅

Reply:
Just realized; the reason I thought it was NRF53X was because somehow my brain concluded nrf52833 was nrf53 series xD; all good; sorry for the confusion.